tests/cpp/test_alias.cpp

```diff
@@ -838,6 +838,7 @@ TEST_F(AliasTest, MergeTwoExpandedBroadcasts) {
   FusionGuard fg(fusion.get());
 
   TensorView* in = TensorViewBuilder()
+                       .ndims(3)
```

Comment:
I removed these ndims calls in https://github.com/NVIDIA/Fuser/pull/2882. A rebase should fix this unless you think there's a good reason to add them back.